box-ui/src/components/ts/JoinMeetingComponent.tsx

```diff
@@ -1,42 +1,65 @@
 import { Box, Button } from '@mui/material';
-import React, { FunctionComponent, useState, useRef, ChangeEvent, MutableRefObject } from 'react';
+import React, { FunctionComponent, useState, ChangeEvent } from 'react';
 import '../css/JoinMeetingComponent.css';
+import { useNavigate } from 'react-router-dom';
 import Keyboard from 'react-simple-keyboard';
 import 'react-simple-keyboard/build/css/index.css';
 import HeaderComponent from './HeaderComponent';
 import OutlinedInput from '@mui/material/OutlinedInput';
 import InputLabel from '@mui/material/InputLabel';
 import FormControl from '@mui/material/FormControl';
 
+interface InputRoom {
+    domain: string;
+    roomName: string;
+}
+
 const JoinMeetingComponent: FunctionComponent = () => {
-    const [meetingName, setMeetingName] = useState('');
-    const [layout, setLayout] = useState('default');
-    const keyboard = useRef();
+    const [inputs, setInputs] = useState<InputRoom>({
+        domain: '',
+        roomName: '',
+    });
+    const [layout, setLayout] = useState<string>('default');
+    const [inputName, setInputName] = useState<string>('default');
+    // const keyboard = useRef();
 
-    const onChange = (input: string): void => {
-        setMeetingName(input);
+    const navigate = useNavigate();
+    const goToLaunchRoom = (): void => {
+        navigate({ pathname: '/launch' }, { replace: true });
```

Comment:
```suggestion
        navigate({ pathname: '/launch' }, { state: { link: `${domain}${roomName}` }, replace: true });
```
Il faut passer le lien au composant non ?

Reply:
yes